File: week-3/day-1/fizzbuzz.py
# /3 fizz
# /5 buzz
# /3 & /5 fizzbuzz
# contain 5 buzzfizz

n = 0
max = int(input('Enter a number: '))

while n < max:
    if n % 3 == 0 and n %5 == 0:
        print(n, ':fizzbuzz')
    elif n % 3 == 0:
        print(n, ':buzz')
    elif n % 5 == 0:
        print (n, ':fizz')
    elif '5' in str(n):
        print(n, ':buzz')
    else:
        print(n)
    n += 1

File: week-7/js-workshop/fizzbuzz.js
'use strict';

var num = 65;
var numbers= [];
var Num = String(num);

for (var i = 0; i < Num.length; i++) {
  numbers.push(Num[i]);
}

for (var j = 0; j < num; j++) {
  if (j % 15 === 0 ||
     (String(j).indexOf('3') > -1 && String(j).indexOf('5') > -1)) {
    console.log('fizzbuzz');
  } else if (j % 5 === 0 || String(j).indexOf('5') > -1) {
    console.log('buzz');
  } else if (j % 3 === 0 || String(j).indexOf('3') > -1) {
    console.log('fizz');
  } else {
    console.log(j);
  }
}


